dev/prologue.jl

```diff
@@ -2,14 +2,19 @@ using P4est_jll: P4est_jll
 export P4est_jll
 
 using ..P4est: _PREFERENCE_LIBP4EST, _PREFERENCE_LIBSC
+using MPIPreferences: MPIPreferences
 
-@static if _PREFERENCE_LIBP4EST == "P4est_jll"
+@static if _PREFERENCE_LIBP4EST == "P4est_jll" && MPIPreferences.binary == "system"
+    @warn "System MPI version detected, but not a system p4est version. To make P4est.jl work, you need to set the preferences."
```

Comment:
Can we please add a link to the docs here?

Reply:
Done in [d0ac85f](https://github.com/trixi-framework/P4est.jl/pull/93/commits/d0ac85f7e659c5e571435e533bcf94a0a24ace82).